rotkehlchen/tests/api/test_tags.py

```diff
@@ -678,3 +683,72 @@ def test_delete_utilized_tag(rotkehlchen_api_server):
     assert len(query) == 1
     assert query[0][0] == f'{SupportedBlockchain.BITCOIN.value}{UNIT_BTC_ADDRESS1}'
     assert query[0][1] == 'public'
+
+
+@pytest.mark.parametrize('number_of_eth_accounts', [0])
+def test_delete_all_tags(rotkehlchen_api_server):
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    tag1 = {
+        'name': 'public',
+        'description': 'My public accounts',
+        'background_color': 'ffffff',
+        'foreground_color': '000000',
+    }
+    response = requests.put(
+        api_url_for(
+            rotkehlchen_api_server,
+            'tagsresource',
+        ), json=tag1,
+    )
+    assert_proper_response(response)
+
+    # Add an account with a tag
+    btc_balances = ['10000']
+    setup = setup_balances(
+        rotki,
+        ethereum_accounts=None,
+        btc_accounts=[UNIT_BTC_ADDRESS1],
+        eth_balances=None,
+        token_balances=None,
+        btc_balances=btc_balances,
+    )
+    with setup.bitcoin_patch:
+        response = requests.put(
+            api_url_for(
+                rotkehlchen_api_server,
+                'blockchainsaccountsresource',
+                blockchain='BTC',
```

Comment:
since you are not testing the btc account creation is this needed? I believe you can manually add the btc address to the `blockchain_accounts` table and assign the already created tag directly there avoiding to use the api and setting the balances.

I believe it should be possible